api/src/test/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceTest.java

```diff
@@ -122,11 +130,16 @@ public void setUp() {
     providedWorkbenchConfig.access.renewal.expiryDays = 365L;
     providedWorkbenchConfig.access.enableComplianceTraining = true;
 
+    providedWorkbenchConfig.absorb.rtTrainingCourseId = RT_ABSORB_COURSE_ID;
+    providedWorkbenchConfig.absorb.ctTrainingCourseId = CT_ABSORB_COURSE_ID;
+
     accessModules = TestMockFactory.createAccessModules(accessModuleDao);
   }
 
   @Test
-  public void testSyncComplianceTrainingStatus() throws Exception {
+  public void testSyncComplianceTrainingStatus_Moodle() throws Exception {
+    providedWorkbenchConfig.absorb.enabledForNewUsers = false;
```

Comment:
is there a way to add that no absorb will be visible

Reply:
I moved the assertions that "validation" entries exist into this test, which I think clarifies that Moodle was used instead of Absorb.